magento/bulk-api-ce
# WebapiAsync Swagger

### Description


Have migrated old commits to new PR pointed to CE, and merged the EE and CE code changes together into one branch. 

Basic integration is working but unable to finish acceptance tests due to outstanding issues in `async-webapi-2`.

**To be completed:**

- [ ] Add nicer Swagger UI paths. Currently like this https://bulk-api/swagger/?type=async
- [x] Finalise actual response description, HTTP response code
- [ ] Add unit tests for `ServiceMetadata` plugin 
- [x] Integration test for schema output 
- [x] Implement final schema URL in tests, blocks (eg `/rest/async/schema/`, `/async/schema` etc)



### Fixed Issues (if relevant)

1. magento-engcom/bulk-api#11 Change Swagger generator

### Manual testing scenarios
1. Visit https://bulk/swagger/?type=async in browser, expect to see async
2. Visit https://bulk/swagger/ in browser, expect to see rest 


### Contribution checklist
